index.html

```diff
@@ -29,7 +29,7 @@
             </li>
             <li class="left-col__menu__item">
               <a href="#" class="left-col__menu__container">
-                <span class="left-col__menu__icon"><i class="fa fa-bell"></i></span>
+                <span class="left-col__menu__icon "><i class="fa fa-bell left-col__menu__icon__notification"><span class="menu__icon__badge" >55</span><span class="menu__icon__badge-plus " >+99</span></i> </span>
```

Comment:
add only one of them and remove that one with a plus

instead use the pseudo class `::before` to add it conditionally